src/app/auth/services/auth.service.ts

```diff
@@ -304,6 +304,13 @@ export const changePassword = async (newPassword: string, currentPassword: strin
   const privateKey = Buffer.from(user.privateKey, 'base64').toString();
```

Comment:
We should use `user.keys.ecc.privateKey` now, otherwise when we deprecate the old fields, this will break